libraries/rush-sdk/src/test/script.test.ts

```diff
@@ -0,0 +1,56 @@
+import * as path from 'path';
+import { Executable } from '@rushstack/node-core-library';
+
+const rushSdkPath: string = path.join(__dirname, '../../lib/index.js');
+const sandboxRepoPath: string = path.join(__dirname, './sandbox');
+const mockPackageFolder: string = path.join(sandboxRepoPath, 'mock-package');
+
+describe('used in script', () => {
+  it('should work when used in script', () => {
+    const result = Executable.spawnSync('node', ['-e', `console.log(require(\"${rushSdkPath}\"))`], {
+      currentWorkingDirectory: mockPackageFolder
+    });
+    expect(result.status).toBe(0);
+    expect(result.stdout.trim()).toMatchInlineSnapshot(`
+      "{
```

Comment:
This test will get broken when the `rush-lib` top-level API changes.

Is this test running `install-run-rush.js`? Tests shouldn't be making network requests.

Reply:
Got it, let me use a mock rush-lib exports instead.